scripts/soak-matrix.sh

```
@@ -0,0 +1,70 @@
+# hosts in one process would share both. Separate processes each get their own statics, and each scenario
+# already targets its OWN database on the shared Postgres server (SoakScenario.DbName), so the parallel
+# runs never collide.
+#
+# Usage:
+#   bash scripts/soak-matrix.sh                 # default 120s window
+#   SOAK_WINDOW_SECONDS=300 bash scripts/soak-matrix.sh
+#
+# Requires Postgres up (bash .claude/start-infra.sh). Exit code is non-zero if ANY scenario fails.
+set -uo pipefail
+
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)